src/Microsoft.DocAsCode.Build.Incrementals/DependencyGraph.cs

```diff
@@ -59,6 +59,13 @@ public IEnumerable<string> Keys
             get { return _dictionary.Keys; }
         }
 
+        public SortedSet<string> GetDirectDependency(string key)
```

Comment:
mean to reuse trygetvalue, but it's ok not reuse it

---
In reply to: [75417844](https://github.com/dotnet/docfx/pull/618#discussion_r75417844) [](ancestors = 75417844,75277009)